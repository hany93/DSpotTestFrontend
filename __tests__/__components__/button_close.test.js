import '@testing-library/jest-dom'
import ButtonClose from '@components/button/button_close'
import { render, screen } from '@testing-library/react'

describe('ButtonClose Component', () => {
	beforeEach(() => {
		render(<ButtonClose />)
	})

	it('Test role img', () => {
		expect(screen.getByRole('img')).toBeInTheDocument()
	})

	it('Test className close', () => {
		expect(document.querySelector('div .close')).toBeInTheDocument()
	})
})
