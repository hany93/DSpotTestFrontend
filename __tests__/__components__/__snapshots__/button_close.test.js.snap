// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonClose Component Test Snapshot 1`] = `
<div>
  <span
    class="close"
  >
    <img
      alt="Photo"
      class=""
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
  </span>
</div>
`;

exports[`ButtonClose Component Test role img and className close 1`] = `
<div>
  <span
    class="close"
  >
    <img
      alt="Photo"
      class=""
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
  </span>
</div>
`;
